src/components/tables/RowActions/Shared/UpdateEntity.tsx

```diff
@@ -49,31 +65,14 @@ function DisableEnableProgress({ enabling, entity, onFinish }: Props) {
                     return failed(draftsResponse);
                 }
 
-                const encryptedEndpointConfig = await encryptConfig(
-                    connectorTag.endpoint_spec_schema,
-                    spec.endpoint.connector.config
-                );
-                if (encryptedEndpointConfig.error) {
-                    return failed(encryptedEndpointConfig);
-                }
-
                 const newDraftId = draftsResponse.data[0].id;
                 setDraftId(newDraftId);
 
-                const draftSpec = generateDraftSpec(
-                    encryptedEndpointConfig.data,
-                    `${entity.connector_image_name}${entity.connector_image_tag}`
-                );
-
-                draftSpec.bindings = spec.bindings ? spec.bindings : [];
-                draftSpec.shards = spec.shards ? { ...spec.shards } : {};
-                draftSpec.shards.disable = !enabling;
-
```

Comment:
This was a huge mistake... we have no reason to mess with the config as deleting just nulls everything and disable/enable updates the `shards` attribute. So no need for messing with config and encryption. Made stuff a lot simpler.